src/js/components/App.js

```diff
@@ -1,39 +1,95 @@
 import MenuInput from './MenuInput.js';
 import MenuList from './MenuList.js';
 import Count from './Count.js';
+import { $ } from '../utils/dom.js';
+import { UPDATE_MSG, DELETE_MSG } from '../constants/index.js';
+import { store } from '../store/index.js';
```

Comment:
```suggestion
import { UPDATE_MSG, DELETE_MSG } from '../constants';
import { store } from '../store';
```

`index.js` 를 지울수도 있겠네용!